src/app/shell/personal-cabinet/shared-cabinet/applications/applications.component.ts

```diff
@@ -80,30 +67,20 @@ export class ApplicationsComponent implements OnInit, OnDestroy, AfterViewInit {
     this.isMobileView = event.outerWidth < 530;
   }
 
-  constructor(
-    protected store: Store,
-    protected router: Router,
-    protected route: ActivatedRoute,
-    protected actions$: Actions
-  ) {
+  constructor(protected store: Store, protected router: Router, protected route: ActivatedRoute, protected actions$: Actions) {
     this.onResize(window);
   }
 
-  onGetApplications(): void {
-    this.getApplications.emit();
-  }
-
   onEntitiesSelect(IDs: string[]): void {
     this.enititiesSelect.emit(IDs);
```

Comment:
should set the first page on the filter change 
![image](https://user-images.githubusercontent.com/27493959/215972460-698b250d-da9c-472b-8aa2-bec06d235e20.png)
![image](https://user-images.githubusercontent.com/27493959/215972514-0a0a2ff9-633c-4332-a2c6-772a27a83d81.png)